public/javascripts/status_script.js

```diff
@@ -1,431 +1,386 @@
-		      "data": document.reader_data[key]['buffs']};
-            yaxis_label = "MB";
-        }
-	series.push(rates);
 
+  // Convert data dict to highcharts format
+  var series = [];
+  var yaxis_label = "";
+  for(var key in document.reader_data){
+    var rates = {};
+    if($("#menu_variable_s").val() == "rate") {
+      rates = {"type": "line", 
+        "name": key+" rate", 
+        "data": document.reader_data[key]['rate']};
+      yaxis_label = "MB/s";
+    } else if($("#menu_variable_s").val() == "buff") {
+      rates = {"type": "area", 
+        "name": key+" buffer", 
+        "data": document.reader_data[key]['buff']};
+      yaxis_label = "MB";
     }
-
-    var chart_opts = {
-        chart: {
-            zoomType: 'xy',
-//            margin: [5, 5, 20, 80],
-        },
-        plotOptions: {
-            series: {
-                fillOpacity: 0.3,
-		lineWidth: 1
-	    },
-	    //line: {
-	//	marker: false
-	 //   },
-        },
-        credits: {
-	    enabled: false,
-        },
-        title: {
-            text: '',
-        },
-        xAxis: {
-            type: 'datetime',
-        },
-        yAxis: {
-            title: {
-                text: yaxis_label,
-	    },	    
-            min: 0,
-	},
-	legend: {
-            enabled: true,
-	},
-	series: series,
-    };
-    var div = 'rate_chart';
-    document.RatePlot = Highcharts.chart(div, chart_opts);
-
+    series.push(rates);
+
+  }
+
+  var chart_opts = {
+    chart: {
+      zoomType: 'xy',
+      //            margin: [5, 5, 20, 80],
+    },
+    plotOptions: {
+      series: {
+        fillOpacity: 0.3,
+        lineWidth: 1
+      },
+    },
+    credits: {
+      enabled: false,
+    },
+    title: {
+      text: '',
+    },
+    xAxis: {
+      type: 'datetime',
+    },
+    yAxis: {
+      title: {
+        text: yaxis_label,
+      },
+      min: 0,
+    },
+    legend: {
+      enabled: true,
+    },
+    series: series,
+  };
+  var div = 'rate_chart';
+  document.RatePlot = Highcharts.chart(div, chart_opts);
 }
 
 function UpdateStatusPage(){
-    var brokers = []; //["fdaq00_broker_0"]; 
-
     UpdateCommandPanel();
     UpdateCrateControllers();
     UpdateFromReaders();
     UpdateCeph();
     UpdateBootstrax();
     UpdateDispatcher();
-    setTimeout(UpdateStatusPage, 5000);
 }
 
 function UpdateDispatcher() {
   $.getJSON("status/get_detector_status?detector=tpc", (data) => {
     if (typeof data.checkin != 'undefined' && data.checkin < 10) {
-      $("#dispatcher_status").html("online");
-      $("#dispatcher_status").css("color", "green");
+      $("#dispatcher_status").html("online").css("color", "green");
     } else {
-      $("#dispatcher_status").html("offline");
-      $("#dispatcher_status").css("color", "red");
+      $("#dispatcher_status").html("offline").css("color", "red");
     }
   });
 }
 
+function UnpackEBStatus(doc) {
+  if (doc.state === 'busy') return ['green', `processing ${doc.runid} on ${doc.max_cores} cores`];
```

Comment:
@darrylmasson could we re-add the targets here or is the tuple of targets confusing?

Reply:
The reason I ask is that e.g. it shows quite nicely which runs are nv/mv etc.

Reply:
This shouldn't be difficult